WcaOnRails/app/views/competitions/new.html.erb

```diff
@@ -2,13 +2,6 @@
 
 <div class="container">
   <h1><%= yield(:title) %></h1>
-
-  <%= simple_form_for @competition, html: { class: 'form-horizontal new-competition' }  do |f| %>
-    <%= render 'shared/error_messages', object: f.object %>
-
-    <%= f.input :name, required: true, autofocus: true %>
-    <%= f.input :competition_id_to_clone, as: :competition_id, only_one: true %>
-
-    <button type="submit" id="create-competition" class="btn btn-success"></button>
-  <% end %>
+  <%= alert :info, 'To clone an existing competition visit the information page for that competition and click the "Clone" link in the menu.', note: true %>
```

Comment:
Nice, anyway I think it shouldn't appear when we are already cloning a competition. So probably it would be better to set this alert in `flash.now` in the *new* action.